Sources/Parsing/Parsers/Whitespace.swift

```diff
@@ -74,12 +85,20 @@ where
 
     @inline(__always)
     func consumeVertical() -> Bool {
+      // Unicode chars from `CharacterSet.newlines`
       switch bytes.first {
-      case .init(ascii: "\n"), .init(ascii: "\r"):
+      case 0xA, 0xB, 0xC, 0xD: // U+000A ~ U+000D
```

Comment:
```suggestion
      case 0xA ... 0xD: // U+000A ~ U+000D
```

Reply:
This change makes the benchmarks fail to build. `bytes.first` is an optional and Swift can't match them with a `ClosedRange<UInt>`. I think its maybe simpler to revert the change. I've just checked that it build and runs the benchmarks without issues after that.